Library/PKPaymentAuthorizationViewController+Helpers.swift

```diff
@@ -0,0 +1,72 @@
+import KsApi
+import PassKit
+
+extension PKPaymentAuthorizationViewController {
+  public static var merchantIdentifier: String {
+    return Secrets.ApplePay.merchantIdentifier
+  }
+
+  public static var allSupportedNetworks: [PKPaymentNetwork] = [
+    .amex,
+    .masterCard,
+    .visa,
+    .discover,
+    .JCB,
```

Comment:
I would just make it `.jcb` 😅

Reply:
This is Apple's choice not ours 😄 it's `PKPaymentNetwork.JCB`

Reply:
lol oops